### PR TITLE
Fix duration

### PR DESCRIPTION
**Pull request by egibbm**
This should solve:
- #85 when RF's generated report provide the starttime/endtime
- approximation starttime/endtime values when report was modified, eg. [by merge](https://groups.google.com/g/robotframework-users/c/TWvZMtbizrs)

### CODE REVIEW

File: robotframework_reportportal/time_visitor.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        if o.starttime:
+            corrected = False
+            for parent_id in _stack:
+                if corrections[parent_id][0] is None or corrections[parent_id][0] > o.starttime:
+                    corrections[parent_id][0] = o.starttime
+                    corrected = True
+            if corrected:
+                logging.debug(
+                    "Correcting parents' starttime to {0} based on {2}={1}".format(o.starttime, o.id, node_class))
+            return
```
Comment (iivanou) [inline robotframework_reportportal/time_visitor.py:26]: It can be omitted.

File: robotframework_reportportal/time_visitor.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        if o.endtime:
+            corrected = False
+            for parent_id in _stack:
+                if corrections[parent_id][1] is None or corrections[parent_id][1] < o.endtime:
+                    corrections[parent_id][1] = o.endtime
+                    corrected = True
+            if corrected:
+                logging.debug(
+                    "Correcting parents' endtime to {0} based on {2}={1}".format(o.endtime, o.id, node_class))
+            return
```
Comment (iivanou) [inline robotframework_reportportal/time_visitor.py:46]: It can be omitted.

File: robotframework_reportportal/time_visitor.py (hunk 1 above)
Comment (egibbm) [reply robotframework_reportportal/time_visitor.py]: done

File: robotframework_reportportal/time_visitor.py (hunk 2 above)
Comment (egibbm) [reply robotframework_reportportal/time_visitor.py]: done

**Review by iivanou — APPROVED** [review]

**Pull request merged**